### PR TITLE
fixe : [Core] Incorrect link to ray.remote package reference

### LINKED ISSUE #42502: [Core] Incorrect link to ray.remote package reference
### Description

The link to `ray.remote` api doc after the first code cell points to the core API main page (https://docs.ray.io/en/latest/ray-core/api/index.html), while it should point to https://docs.ray.io/en/latest/ray-core/api/doc/ray.remote.html

### Link

https://docs.ray.io/en/latest/ray-core/tasks.html#ray-remote-functions

### PR DESCRIPTION
**Pull request by geoffreylgv**
- Incorrect link to ray.remote package reference
- Closes #42502

## Checks

- [x] I've signed off every commit(by using the -s flag, i.e., `git commit -s`) in this PR.
- [ ] I've run `scripts/format.sh` to lint the changes in this PR.
- [ ] I've included any doc changes needed for https://docs.ray.io/en/master/.
    - [ ] I've added any new APIs to the API Reference. For example, if I added a 
           method in Tune, I've added it in `doc/source/tune/api/` under the 
           corresponding `.rst` file.
- [ ] I've made sure the tests are passing. Note that there might be a few flaky tests, see the recent failures at https://flakey-tests.ray.io/
- Testing Strategy
   - [ ] Unit tests
   - [ ] Release tests
   - [ ] This PR is not tested :(

### CODE REVIEW

**Comment by geoffreylgv** [comment]
Hi @angelinalg, @can-anyscale, @ericl, @robertnishihara, @rkooo567, @krfricke 
Please, can you review this PR
Thanks

**Comment by geoffreylgv** [comment]
> Hi @angelinalg, @can-anyscale, @ericl, @robertnishihara, @rkooo567, @krfricke Please, can you review this PR Thanks

@angelinalg, @rickyyx @architkulkarni  @aslonnie 

Any updates ?

**Review by aslonnie — APPROVED** [review]
some one from @ray-project/ray-docs will review this. they are the code owners.

please do not ping me or @can-anyscale next time.

**Comment by geoffreylgv** [comment]
> some one from @ray-project/ray-docs will review this. they are the code owners.
> 
> please do not ping me or @can-anyscale next time.

Got it, thanks

File: doc/source/ray-core/tasks.rst [hunk 1]
```diff
@@ -14,7 +14,7 @@ Ray enables arbitrary functions to be executed asynchronously on separate Python
             :start-after: __tasks_start__
             :end-before: __tasks_end__
 
-        See the `ray.remote package reference <package-ref.html>`__ page for specific documentation on how to use ``ray.remote``.
+        See the `ray.remote package reference <ray.remote.html>`__ page for specific documentation on how to use ``ray.remote``.
```
Comment (aslonnie) [inline doc/source/ray-core/tasks.rst:17]: I think it needs to be `./api/doc/ray.remote.html`

this package is rendered at `/en/master/ray-core/tasks.html`, and the target is at `/en/master/ray-core/api/doc/ray.remote.html`

Comment (aslonnie) [reply doc/source/ray-core/tasks.rst:17]: there are also many other references to `ray-core/package-ref.html` in the code base.

maybe the right thing is to add back `package-ref.html` with a redirect.

@angelinalg @simran-2797 do you know how `package-ref.html` gets lost?

Comment (angelinalg) [reply doc/source/ray-core/tasks.rst:17]: Thanks for investigatin, @aslonnie. The `package-ref.html` reference was a mystery to me. I wonder if @peytondmurray understands it.

**Pull request closed without merging**